esm-collection/json-schema/schema.json

```diff
@@ -1,14 +1,12 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "schema.json#",
   "title": "ESM Extension Specification",
   "description": "This object represents Collections in a Earth System Model Catalog.",
   "allOf": [
     {
```

Comment:
I looked at the other extenal extension again: https://github.com/radiantearth/stac-spec/issues/874#issuecomment-675683146

There, I don't refer to the actual specs. The disadvantage with the links/schemas is that they are bound to a specific version and won't work for any future version unless the links are changed here as well.

Maybe the best way is to remove them and add the required collection-assets extension to stac_extensions in the schema. That way validators should handle it correctly.
The issue with the links